src/Transition/CreateNewIDFUsingRulesV8_3_0.f90

```diff
@@ -378,7 +378,9 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                 ! Remove Max flow rate field
                 nodiff=.false.
                 CALL GetNewObjectDefInIDD(ObjectName,NwNUmArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
-                OutArgs(1:10) = InArgs(1:10)
+                OutArgs(1:3) = InArgs(1:3)
+                OutArgs(4)='Design Flow Rate'
```

Comment:
@mitchute This line is filling OutArgs(4) with a string which fails.  Should be `OutArgs(4) = InArgs(11)` if I'm following this correctly - moving arg11 to be arg4.